typings/tests/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+// This additional tsconfig is required by dtslint.
```

Comment:
Can we make this more detailed as to why this file is required - also, did you check in on the ts repo?

Reply:
Added: 
```// This additional tsconfig is required by dtslint. It is used to configure
// tslint, which is used to compile the tests.```

I checked the repo, and through looking at the source code it seems it is used to configure tslint and to compile the tests.